cmd/kubelet/app/options/options.go

```diff
@@ -491,6 +491,7 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.StringVar(&c.CgroupDriver, "cgroup-driver", c.CgroupDriver, "Driver that the kubelet uses to manipulate cgroups on the host.  Possible values: 'cgroupfs', 'systemd'")
 	fs.StringVar(&c.CgroupRoot, "cgroup-root", c.CgroupRoot, "Optional root cgroup to use for pods. This is handled by the container runtime on a best effort basis. Default: '', which means use the container runtime default.")
 	fs.StringVar(&c.CPUManagerPolicy, "cpu-manager-policy", c.CPUManagerPolicy, "CPU Manager policy to use. Possible values: 'none', 'static'.")
+	fs.Var(cliflag.NewMapStringString(&c.CPUManagerPolicyOptions), "cpu-manager-policy-options", "A set of key=value CPU Manager policy options to use, to fine tune their behaviour. If not supplied, keep the default behaviour.")
```

Comment:
`NewMapStringStringNoSplit` seems the right thing here.

Reply:
To elaborate a bit: I don't see values including `=` or `,` on the horizon, but since this is already handled by the utilities and it seems more future proof I'm leaning towards using `NewMapStringStringNoSplit`. @klueska any preference from your side?